src/components/MarkdownOutput/hooks/__test__/useMarkdownToReact.spec.js

```diff
@@ -0,0 +1,26 @@
+import { renderHook } from '@testing-library/react-hooks';
+import ReactDOMServer from 'react-dom/server';
+import useMarkdownToReact from '../useMarkdownToReact';
+
+describe('useMarkdownToReact', () => {
+    it('should return valid html', () => {
+        const values = [
+            '# This is a level 1 header',
+            '## This is a level 2 header',
+            '### This is a level 3 header',
+            'This is a paragraph',
+            '-----',
```

Comment:
I like these, can we add more? it could be in differents tests that check different markup generation, like table, code,  ...